### PR TITLE
Javascript3 Week2 HW

### PR DESCRIPTION
**Pull request by Saikiruthiga**
fetch function changed to async await approach
Added score details.

### CODE REVIEW

File: quizApp.js [hunk 1]
```diff
@@ -222,18 +221,50 @@ function doSorting(type) {
...
+  } catch (error) {
+    console.log(error);
+  }
+
   displayQuestion(questionArray, true);
 }
+function calcScore() {
+  let score = 0;
+  const questions = document.querySelectorAll("#quiz-questions > div");
+  questions.forEach((question) => {
```
Comment (babak-f) [inline quizApp.js:246]: Try adding empty lines to make the code easier to read :)

File: quizApp.js [hunk 2]
```diff
@@ -56,17 +55,16 @@ function startQuiz() {
...
 
       const correctIcon = document.createElement("i");
-      correctIcon.className = `fa fa-check-circle scorebtn-${index}`;
+      correctIcon.className = `fa fa-check-circle`;
       correctIcon.id = `icon-correct${index}`;
       playerContainer.appendChild(correctIcon);
 
       const wrongIcon = document.createElement("i");
-      wrongIcon.className = `fa fa-times-circle scorebtn-${index}`;
+      wrongIcon.className = `fa fa-times-circle`;
```
Comment (babak-f) [inline quizApp.js]: You can just use a single quote notation (') here. Can you guess why? :)

**Review by babak-f — APPROVED** [review]
Good job :+1:

File: quizApp.js (hunk 2 above)
Comment (Saikiruthiga) [reply quizApp.js]: Yes, I will not add any values here. so i need not use template literal so i can use single quote or double quote.

File: quizApp.js (hunk 1 above)
Comment (Saikiruthiga) [reply quizApp.js:246]: Sure, I keep this in mind.

**Pull request merged**